test/test.bctklib/ReadOnlyStoreTests.cs

```diff
@@ -63,7 +63,6 @@ public void checkpoint_settings()
     [Fact]
     public void checkpoint_store_throws_on_incorrect_metadata()
     {
-        Assert.Throws<Exception>(() => new CheckpointStore(checkpointFixture.CheckpointPath, network: 0));
```

Comment:
any reason for this to be removed?

Reply:
Aha, this line of testing fails. I think it's a bug for UT, not neo code.